tools/testsys/src/aws_resources.rs

```diff
@@ -391,11 +391,9 @@ impl AwsEcs {
         testsys_images: &TestsysImages,
     ) -> Result<Vec<Crd>> {
         match test {
-            TestType::Conformance => {
-                return Err(anyhow!(
-                    "Conformance testing for ECS variants is not supported."
-                ))
-            }
+            TestType::Conformance => Err(anyhow!(
```

Comment:
Doesn't seem like this change is related to the other commits in the series :stuck_out_tongue:, should it be part of a different PR?

Reply:
It's related to the update to Rust 1.63.0 in the actions workflow, but it looks like @stmcginnis is fixing it in #2443. I'll rebase and drop the patch when that's merged.